packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/en.ftl

```diff
@@ -14,3 +14,15 @@ sub-change-submit = Confirm change
 sub-update-current-plan-label = Current plan
 sub-update-new-plan-label = New plan
 sub-update-total-label = New total
+sub-update-prorated-upgrade = Prorated Upgrade
+
+## Product name and frequency of payment for subscription plan change
+
+# Variables:
+#   $name (String) - Name of the upgraded product
+sub-update-new-plan-daily = {$name} (Daily)
+sub-update-new-plan-weekly = {$name} (Weekly)
+sub-update-new-plan-monthly = {$name} (Monthly)
+sub-update-new-plan-yearly = {$name} (Yearly)
```

Comment:
```suggestion
sub-update-new-plan-daily = { $name } (Daily)
sub-update-new-plan-weekly = { $name } (Weekly)
sub-update-new-plan-monthly = { $name } (Monthly)
sub-update-new-plan-yearly = { $name } (Yearly)
```

Fluent variables should have spaces separating it from the brackets.

Nit: It'd be nicer if the variable name were more descriptive (e.g. $productName instead of $name), again this would be for localizers to better picture how the string would appear.

Reply:
Agreed. I was originally following earlier variable naming for consistency but I much prefer a more descriptive variable name myself. Updated to `$productName`. 😄 Let me know!